StorageUtility/Windows/StorageUtility.cpp

```diff
@@ -203,6 +203,30 @@ eErrorCode GetStorageAdapterProperty( HANDLE Handle, sStorageAdapterProperty& Ad
     return( eErrorCode::None );
 }
 
+eErrorCode GetPhysicalDiskNumber(HANDLE Handle, U32& PhysicalDiskNumber)
```

Comment:
@MaiChiemMinh: With the needs of distinguishing drives while firmware download, the drive needs an unique identification so physical disk number was born.

@Dien: why use HANDLE& ?, this function does not change the HANDLE